src/sources/internal_metrics.rs

```diff
@@ -29,7 +29,7 @@ pub struct InternalMetricsConfig {
 
     /// Overrides the default namespace for the metrics emitted by the source.
     ///
-    /// By default, `vector` is used.
+    /// Overrides the default namespace.
```

Comment:
I wasn't sure what would be best here, so am open to suggestions! I actually think the line above would work well, but that doesn't seem to get pulled into the docs. https://a.cl.ly/rRu5bEEp

Reply:
This seems like the best for now, but we will probably want to inject the name here as the internal metrics will actually be prefixed with `vector_` otherwise. I'll make a note of that for the additional work coming up next quarter.